.github/workflows/publish.yaml

```diff
@@ -0,0 +1,48 @@
+name: publish
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  publish:
+    strategy:
+      matrix:
+        go-version: [1.13.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@master
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }} 
+      - name: Docker meta
+        id: meta
+        uses: crazy-max/ghaction-docker-meta@v1
```

Comment:
I might be wrong, but i thought we are not allowing external actions.

Reply:
Actually, I took this piece of code from the sample which is made by @LucasRoesler in the certifier project, https://github.com/LucasRoesler/certifier/actions/runs/377326951/workflow

Reply:
thats not been merged as far as i can see, the approach in https://github.com/openfaas/faas-cli/blob/master/.github/workflows/publish.yaml#L39-L54 seems like what we have settled on

Reply:
Should I update my like the example above?